eclipse/site/pom.xml

```diff
@@ -171,7 +171,7 @@
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>${jbossTychoPluginsVersion}</version>
+        <version>0.23.1</version>
```

Comment:
The jboss-tycho-version is currently set to 0.26.0, the creation of the TP is nto possible, we need to use 0.26.0-SNAPSHOT, I asked on hipchat if the tag is planned.
My recommendation until then:
- use jboss-tycho-version for repository-utils too
- set property value to 0.26.0-SNAPSHOT